src/Auth/UsersController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Canvas\Auth;
+
+use Canvas\Auth\Models\Companies;
+use Canvas\Auth\Models\Users;
+use Baka\Contracts\Http\Api\CrudBehaviorTrait;
+use Baka\Http\Api\BaseController;
+use Exception;
+use Phalcon\Http\Response;
+
+/**
+ * Base controller.
+ *
+ */
+abstract class UsersController extends BaseController
```

Comment:
@rwhite27 same thing trait or if not we can just move all these functions to the actual user controller and no need to extend